src/app/cli/commands.ts

```diff
@@ -0,0 +1,729 @@
+                );
+                console.log(
+                    `  Session TTL: ${chalk.cyan(config.sessions?.sessionTTL ? `${config.sessions.sessionTTL / 1000}s` : 'Default')}`
+                );
+
+                // MCP Servers
+                console.log(chalk.bold('\n🔌 MCP Servers:'));
+                const servers = Object.keys(config.mcpServers || {});
+                if (servers.length > 0) {
+                    for (const server of servers) {
+                        console.log(`  ${chalk.cyan(server)}`);
+                    }
+                } else {
+                    console.log(chalk.dim('  No MCP servers configured'));
+                }
+
+                console.log();
+            } catch (error) {
+                logger.error(
+                    `Failed to get configuration: ${error instanceof Error ? error.message : String(error)}`
+                );
+            }
+            return true;
+        },
+    },
+    {
+        name: 'stats',
+        description: 'Show system statistics',
+        usage: '/stats',
+        handler: async (_args: string[], agent: SaikiAgent) => {
+            try {
+                console.log(chalk.blue('\n📊 System Statistics:\n'));
+
+                // Session stats
+                const sessionStats = await agent.sessionManager.getSessionStats();
+                console.log(chalk.bold('💬 Sessions:'));
+                console.log(
+                    `  Total Sessions: ${chalk.cyan(sessionStats.totalSessions.toString())}`
+                );
+                console.log(`  In Memory: ${chalk.cyan(sessionStats.inMemorySessions.toString())}`);
+                console.log(`  Max Allowed: ${chalk.cyan(sessionStats.maxSessions.toString())}`);
+
+                // MCP stats
+                console.log(chalk.bold('\n🔌 MCP Servers:'));
+                const connectedServers = agent.mcpManager.getClients().size;
+                const failedConnections = Object.keys(
+                    agent.mcpManager.getFailedConnections()
+                ).length;
+                console.log(`  Connected: ${chalk.green(connectedServers.toString())}`);
+                if (failedConnections > 0) {
+                    console.log(`  Failed: ${chalk.red(failedConnections.toString())}`);
+                }
+
+                // Tools
+                try {
+                    const tools = await agent.mcpManager.getAllTools();
+                    console.log(
+                        `  Available Tools: ${chalk.cyan(Object.keys(tools).length.toString())}`
+                    );
+                } catch {
+                    console.log(`  Available Tools: ${chalk.dim('Unable to count')}`);
+                }
+
+                console.log();
+            } catch (error) {
+                logger.error(
+                    `Failed to get statistics: ${error instanceof Error ? error.message : String(error)}`
+                );
+            }
+            return true;
+        },
```

Comment:
this is great! let's also add a `tools` command that simply lists out all the tools

could also add `prompt` to list the current system prompt